src/multivariate/dirichletmultinomial.jl

```diff
@@ -0,0 +1,114 @@
+immutable DirichletMultinomial{T <: Real} <: DiscreteMultivariateDistribution
+    n::Int
+    α::Vector{T}
+    α0::T
+
+    function DirichletMultinomial(n::Integer, α::Vector{T})
+        α0 = sumabs(α)
+        sum(α) == α0 || throw(ArgumentError("DirichletMultinomial: alpha must be a positive vector."))
+        n > 0 || throw(ArgumentError("DirichletMultinomial: n must be a positive integer."))
```

Comment:
Oh I didn't realize it was doing that. You can leave `Dirichlet` as-is and we can revisit that another time.